assigner/config/versions.py

```diff
@@ -0,0 +1,48 @@
+import jsonschema
+import logging
+
+from assigner.config.schemas import SCHEMAS
+from assigner.config.upgrades import UPGRADES
+
+logger = logging.getLogger(__name__)
+
+class ValidationError(jsonschema.ValidationError):
+    pass
+
+
+def validate(config, version=(len(SCHEMAS) - 1)):
+    assert version < len(SCHEMAS)
```

Comment:
I've never seen this. I noticed it was a builtin but this makes sense.

Reply:
Yeah! I should rename this to `_validate` or rewrite this to throw...going to do the former.